src/logic/puzzles.js

```diff
@@ -2500,228 +6576,2618 @@ export const puzzles = [
-    winText: "Impressive work!",
+      "If you need inspiration, just think about that that time I double crossed you!",
+    winText:
+      "Nice job! Let us know what you thought of the game at www.patreon.com/c/skedwards88. We’re still in beta, so all feedback is appreciated!",
     robotStartMood: "happy",
     robotEndMood: "happy",
     puzzle: [
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.portal,
-      features.basic,
-      features.basic,
-      features.basic,
-      features.basic,
-      features.jet,
-      features.outer,
-      features.terminal1,
-      features.exit,
-      features.terminal3,
-      features.flask,
-      features.portal,
-      features.basic,
-      features.outer,
-      features.basic,
-      features.basic,
-      features.basic,
-      features.portal,
-      features.flask,
-      features.basic,
-      features.outer,
-      features.basic,
-      features.flask,
-      features.portal,
-      features.basic,
-      features.terminal2,
-      features.basic,
-      features.outer,
-      features.basic,
-      features.portal,
-      features.flask,
-      features.terminal4,
-      features.start,
-      features.basic,
-      features.outer,
-      features.terminal5,
-      features.basic,
-      features.basic,
-      features.basic,
-      features.basic,
-      features.portal,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
-      features.outer,
+      "outer",
+      "outer",
+      "outer",
+      "outer",
+      "outer",
+      "portal",
+      "outer",
+      "outer",
+      "portal",
+      "outer",
+      "outer",
+      "outer",
+      "flask",
+      "outer",
+      "portal",
+      "5",
+      "basic",
+      "3",
+      "basic",
+      "basic",
+      "portal",
+      "outer",
+      "basic",
+      "outer",
+      "basic",
+      "outer",
+      "exit",
+      "outer",
+      "outer",
+      "basic",
+      "outer",
+      "basic",
+      "outer",
+      "basic",
+      "outer",
+      "portal",
+      "flask",
+      "basic",
+      "jet",
+      "basic",
+      "4",
+      "portal",
+      "outer",
+      "basic",
+      "outer",
+      "basic",
+      "outer",
+      "basic",
+      "outer",
+      "portal",
+      "1",
+      "basic",
+      "2",
+      "basic",
+      "basic",
+      "start",
+      "outer",
+      "portal",
+      "outer",
+      "portal",
+      "outer",
+      "portal",
+      "outer",
+    ],
+  },
+  {
+    station: "Dial Up Station",
+    room: "1",
+    startingText: "Back in my day, we had these ‘phones’ with numbers on them…",
```

Comment:
Can we do like this?
```suggestion
    startingText: "Back in my day, we had these 'phones' with numbers on them...",
```